### PR TITLE
USSD registration send dialback sms

### PR DESCRIPTION
**Pull request by gsvr**

### CODE REVIEW

**Comment by gsvr** [comment]
@imsickofmaps @NicolaasDieBaas Please review.

File: go-sms_inbound.js [hunk 1]
```diff
@@ -619,23 +619,44 @@ go.utils = {
...
+    },
 
-        return api.put(im.config.vumi_http.url, {
-            data: {
-                "to_addr": go.utils.normalize_msisdn(to_addr, '234'),  // nigeria
-                "content": sms_message
-            }
-        });
+    send_text: function(im, user_id, sms_content) {
+        var payload = {
```
Comment (imsickofmaps) [inline go-sms_inbound.js]: Add a comment as to why `.context` not being used

File: src/utils.js [hunk 2]
```diff
@@ -612,23 +612,44 @@ go.utils = {
...
+    },
 
-        return api.put(im.config.vumi_http.url, {
-            data: {
-                "to_addr": go.utils.normalize_msisdn(to_addr, '234'),  // nigeria
-                "content": sms_message
-            }
-        });
+    send_text: function(im, user_id, sms_content) {
+        var payload = {
```
Comment (imsickofmaps) [inline src/utils.js:642]: Add a comment as to why `.context` not being used

**Comment by imsickofmaps** [comment]
:+1:

**Pull request merged**